ui/app/src/Updates.elm

```diff
@@ -96,6 +99,64 @@ update msg model =
             in
                 ( { model | filter = { filter | text = t } }, Cmd.none )
 
+        KeyDownMsg code ->
+            let
+                mode =
+                    if List.member code modifierKeys then
+                        Append
+                    else
+                        Replace
+            in
+                ( { model | mode = mode }, Cmd.none )
+
+        KeyUpMsg code ->
+            let
+                mode =
+                    if List.member code modifierKeys then
+                        Replace
+                    else
+                        Append
+            in
+                ( { model | mode = mode }, Cmd.none )
+
+        AddLabel msg ( key, value ) ->
```

Comment:
This is only relevant for the `SilenceList` and the `AlertList` right? Would it be possible to move it down into the respective `update` functions? Feels weird that this lives in the global `update` function.